src/main/java/org/sunbird/user/registration/service/UserRegistrationServiceImpl.java

```diff
@@ -588,4 +621,29 @@ private Boolean isApprovedDomains(String emailDomain, String domainType) {
 				Constants.KEYSPACE_SUNBIRD, Constants.TABLE_MASTER_DATA, propertyMap, Arrays.asList(Constants.CONTEXT_TYPE, Constants.CONTEXT_NAME));
 		return CollectionUtils.isNotEmpty(listOfDomains);
 	}
+
+	private List<String> getApprovedDomainsFromDB() {
```

Comment:
what will happen if the domain exist in both list?

Reply:
Updated code to tackle this